ProcessLib/HeatTransportBHE/LocalAssemblers/HeatTransportBHELocalAssemblerBHE.h

```diff
@@ -29,14 +29,13 @@ class HeatTransportBHELocalAssemblerBHE
     : public HeatTransportBHELocalAssemblerInterface
 {
     static constexpr int bhe_unknowns = BHEType::number_of_unknowns;
-    static constexpr int single_bhe_unknowns_size = ShapeFunction::NPOINTS;
-    static constexpr int temperature_size = ShapeFunction::NPOINTS;
-    static constexpr int temperature_index = 0;
-    static constexpr int bhe_unknowns_size =
-        single_bhe_unknowns_size * bhe_unknowns;
+    static constexpr int n_int_points = ShapeFunction::NPOINTS;
```

Comment:
I'm probably missing something, but the number of integration points is not equal to the number of shape functions in general.

Reply:
Yes, but in BHE elements, which are line elements only. And there is no need to use higher order shape function considering simulation purpose. So the intergration point here in BHE line elements should be 2. For better readability, we changed the `unknowns_size` to the `n_int_points`. If you insist, I can also remove this change.

Reply:
The number of integration points is independent of the element type.
[integration_order](https://doxygen.opengeosys.org/ogs_file_param__prj__processes__process__integration_order.html)
[shape function order](https://doxygen.opengeosys.org/ogs_file_param__prj__process_variables__process_variable__order.html)
Both can be set through the project file. So you can have linear line elements with fourth integration order. This is used *e.g.* in richards-mechanics, where there is high non-linearity in certain setups.